Tests/PackageLoading/ConventionTests.swift

```diff
@@ -99,17 +99,17 @@ class ConventionTests: XCTestCase {
     func testCInTests() throws {
```

Comment:
I think tests should also cover the error cases `Sources` has a test module and `Tests` has a non-test module